### PR TITLE
splash screen, footer page, and added new questions

### PR DESCRIPTION
**Pull request by varmaa23**
added a png splash screen, rerouted 'share' button on footer page to messages app, and added 3 new questions to dance challenge. i still haven't resolved the merge conflict bc my machine isn't allowing me to view the merge conflicts. can someone else fix these

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -52,44 +55,57 @@ const HomeScreen = ({ navigation }) => {
...
+                        source={require('./assets/genconnect_logo-black.png')}
+                        style={{
+                            width: 350,
+                            height: 130,
+                            marginLeft: 10,
+                            marginRight: 10
+                        }}
+                    />
+                </StyledLogo>
+                <StyledTagline style={{ paddingTop: 10, paddingBottom: 20 }}>The game that gets families talking!</StyledTagline>
```
Comment (sukritsangvong) [inline App.js:69]: should avoid in-line styling since we committed to using `styled-component`

File: App.js [hunk 2]
```diff
@@ -52,44 +55,57 @@ const HomeScreen = ({ navigation }) => {
...
+                    />
+                </StyledHomeButtonView>
+                <StatusBar style="auto" />
+                <TouchableOpacity
+                    onPress={() => navigation.navigate('Information')}
+                    style={{
+                        position: 'absolute',
+                        right: 20,
+                        top: 40,
+                    }} >
```
Comment (sukritsangvong) [inline App.js:100]: same here

File: App.js [hunk 3]
```diff
@@ -52,44 +55,57 @@ const HomeScreen = ({ navigation }) => {
...
+                    <Image
+                        source={require('./assets/footer_info.png')}
+                        style={{
+                            width: 50,
+                            height: 50
+                        }}
```
Comment (sukritsangvong) [inline App.js:106]: and here

Comment (sukritsangvong) [reply App.js:106]: but yea can just leave it like this since we are close to the deadline. should avoid this in the future

File: app.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "icon": "./assets/icon.png",
     "splash": {
       "image": "./assets/splash.png",
-      "resizeMode": "contain",
+      "resizeMode": "cover",
```
Comment (sukritsangvong) [inline app.json:10]: what does this do? is this related to iPad sizing?

File: components/HomeButton.js [hunk 5]
```diff
@@ -35,10 +34,10 @@ const styles = StyleSheet.create({
         textAlign: 'center',
         fontSize: 30,
         minWidth: 165,
-        minHeight: 220,
+        minHeight: 20,
         padding: 10,
         left: 5,
-        top: 85,
+        top: 55,
```
Comment (sukritsangvong) [inline components/HomeButton.js:40]: have u checked to make sure that all these changes do not affect other screen sizes? changing existing values by a lot seems alarming

File: App.js (hunk 2 above)
Comment (varmaa23) [reply App.js:100]: i don't think i did this one, what's the best way to resolve this @sukritsangvong

File: app.json (hunk 4 above)
Comment (varmaa23) [reply app.json:10]: this is for the splash screen. it makes sure that there's no ugly white background when it appears @sukritsangvong

File: components/HomeButton.js (hunk 5 above)
Comment (varmaa23) [reply components/HomeButton.js:40]: yes, looks good on bigger and smaller screens.
